### ISSUE TITLE
Controls are not showing on Android for vimeo player

### ISSUE THREAD

**Issue by gboston** [sampotts/plyr#2018]
When opening up the following page: https://plyr.io/#vimeo and clicking play

# Expected behaviour
Clicking video should show video controls

### Actual behaviour
Not able to click pause or show controls

### Steps to reproduce
1. Open https://plyr.io/#vimeo
2. Click play

### Environment

- Browser: Chrome
- Version: LTS
- Operating System: Android
- Version: 10

**Comment by evild70** [comment]
Same thing happens in: 
### Environment

* Browser: Chrome & Firefox
* Operating System: Android
* Version: 11

**Comment by michaelfigment** [comment]
https://plyr.io/#vimeo

Also on Android Opera (Full version). 

Youtube, Video & Audio all work fine on the demo.

**Comment by PGEmerson-Dev** [comment]
Same thing happens in iOS

**Comment by lmcsu** [comment]
I check every day if the issue is resolved, seems like it's not going to be yet.
Any workarounds maybe? How to fix that manually?

**Comment by michaelfigment** [comment]
I reverted to the previous release, as the bug isn't present in that
version.

**Comment by lmcsu** [comment]
@michaelfigment thank you, I downgraded to version 3.6.2 and it works.

**Comment by ImrozCinchy** [comment]
Downgrading to plyr 3.6.2 is also not working. Using ngx-plyr 4.0.0. Any updates on this issue?

**Comment by tomferrall** [comment]
Has anyone come up with a fix at all?

I had previously downgraded to 3.6.2 but it suddenly stopped working (not playing at all) so I figured a player that at least works in most instances is something - so I'm using 3.6.4.

Such a shame as I'd really like to use this player!

**Comment by alexvibealex** [comment]
Yes, we are getting the same issue on chrome on android after updating plyrjs to 3.6.4.
After pausing the video the controls are not shown.

**Comment by alexvibealex** [comment]
Hi anyone with some urgent fix , users are not able to see any controls after playing vimeo video  with new version 3.6.4??

**Comment by VictorSholomickiy** [comment]
Hello, any updated or maybe workarounds?

**Comment by mezpleo** [comment]
I'm having this same issue, anyone found a solution?
Once I press play, the controls disappear and won't reappear again unless I press space bar

Happens on both:
Chrome & Firefox
